releases/cloud/point-2023-09-19-release-note.md

```diff
@@ -0,0 +1,20 @@
+---
+description: >-
+  This article lists improvements and bug fixes in the Syskit Point Cloud
+  version 2023.4.0.XX
+---
+
+# September 19, 2023
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+## Bug Fixes
+
+* **Fixed an issue** where the Delete action was not visible on the Inactive Guest User screen when drilled from the Security & compliance dashboard.
+* **Fixed a bug** where the Automated column showed incorrect values for some policies on the Policies settings screen.
```

Comment:
show je irregular verb, nisam siguran je li showed točno, (mislim da je show>shew>shown)